great-idea/index.html

```diff
@@ -1,23 +1,121 @@
-<!doctype html>
-
-<html lang="en">
 <head>
-  <meta charset="utf-8">
-
   <title>Great Idea!</title>
+  <link rel="stylesheet" href="css/index.css" />
+</head>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+<!--
+  -<link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <!---<link rel="stylesheet" href="css/index.css">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+  <![endif]
+-->
 
+<html>
   <body>
+    <nav>
+      <a id="nav" href="./services.html">Services</a>
+      <a id="nav" href="#">Product</a> <a id="nav" href="#">Vision</a>
+      <a id="nav" href="#">Features</a> <a id="nav" href="#">About</a>
+      <a id="nav" href="#">Contact</a>
+      <img
+        class="logo"
+        src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png"
+        alt="Great Idea! Company logo."
+      />
+    </nav>
+
+    <div class="header">
+      <section class="sectionHeader">
+        <pre id="title">
+Innovation
+On
+Demand</pre
+        >
+        <a href="./services.html" id="btnT"> Get Started</a>
```

Comment:
I know what you tried doing here, but try to avoid using anchor tags without a ```nav``` tag as a wrapper